src/demo/DemoTestLoanService.java

```diff
@@ -2452,10 +2452,10 @@ public static void testAddAndRemoveSetllementAccounts() throws MambuApiException
 		System.out.println("Obtaining the product type of the loan account");
 		LoanProduct loanProduct = loanService.getLoanProduct(productTypeKey);
 
-		if (!loanProduct.isAccountLinkingEnabled()) {
+		if (!loanProduct.isAccountLinkingEnabled() || loanProduct.isAutoCreateLinkedAccounts()) {
```

Comment:
Do we need to check for both? Is it possible to have isAutoCreateLinkedAccounts() =true when loanProduct.isAccountLinkingEnabled()= false?